### ISSUE TITLE
[Feature Request]: Get info only for specific train

### ISSUE THREAD

**Issue by rschueler** [FaserF/ha-deutschebahn#10]
### The request

It would be helpful if the sensor could be configured to only provide information for a specific train or connection, such as one defined by the time of departure or train number.

As I take the same train every morning, it would be beneficial for me to know as early as possible if there are any delays or cancellations for my planned train. With the current setup, I can only see the next upcoming connection, but the sensor may not pick up information about later trains that have been cancelled. 

I would like to create an automation that checks if my desired train is on time and, if not, have my smart home system wake me up earlier.

**Comment by andgolearning** [comment]
Hello all, can I also track multiple trains in the HACS version? In other words, create several sensors? If so, please give me a tip on how this can be done: Many greetings

**Comment by FaserF** [comment]
@andgolearning yes, just add a second sensor by following the steps from here again:
https://github.com/FaserF/ha-deutschebahn#configuration

**Comment by FaserF** [comment]
about the feature request, I will take a look into it.
Most likely this would be easier archived with the new HaFAS intergration, once it gets merged from Homeassistant as a official addon.

**Comment by umrath** [comment]
> @andgolearning yes, just add a second sensor by following the steps from here again: https://github.com/FaserF/ha-deutschebahn#configuration

But this does not work for the same route. You can only have one sensor per route.

**Comment by KreppAE** [comment]
That would be incredibly helpful to have it pull the information for a specific train

**Comment by fastlane086** [comment]
> > @andgolearning yes, just add a second sensor by following the steps from here again: https://github.com/FaserF/ha-deutschebahn#configuration
> 
> But this does not work for the same route. You can only have one sensor per route.

So here's how I managed to add the same connection twice.

My connection is Hirschaid - Nuremberg. I have entered once as connection:

From: Hirschaid
To: Nuremberg

and the other

From: Hirschaid
To: Nuremberg main station
Offset in minutes: 30

It is enough to change the name of the station a little bit to add it.

The best is just to try it out

**Comment by FaserF** [comment]
Done with Version 2.1.0

**Comment by umrath** [comment]
This does not show up for me.
I have 2.1.0 installed (and rebooted), but do not see this dialogue. It very much looks like 2.0.x to me still.
Am I missing something?

![Bildschirmfoto 2023-11-30 um 16 59 11](https://github.com/FaserF/ha-deutschebahn/assets/1010437/7f9b1907-f7ce-4b54-a64d-60d8d4290c8a)

**Comment by FaserF** [comment]
Oh my fault, this is not finished yet, I mixed up the Github issue
